### ISSUE TITLE
Serial Number: Manditory by Asset Model

### ISSUE THREAD

**Issue by jmartins1** [grokability/snipe-it#8858]
Would it be possible to have the serial number field check for duplicates; similar to the Asset Tag field?

Updated request.

**Comment by LordDamocles** [comment]
I'd just put 'temp' or 'xx' in the SN fields that you don't need.

**Comment by jmartins1** [comment]
Yes, but I'd like to also prevent duplicate serial numbers from accidently bring entered.

**Comment by whale-food** [comment]
I would like to +1 this. Our company uses Jamf Pro and jamf2snipe to sync over serial numbers, names, etc. from our computers in Jamf. This doesn't stop the occasional instance of someone adding a computer to SnipeIT manually though and not including the serial of the computer. Making the serial field required for certain models would be perfect for us.

**Comment by snipe** [comment]
Added in https://github.com/grokability/snipe-it/pull/16947
